service/src/main/java/io/camunda/service/CamundaServices.java

```diff
@@ -52,10 +52,6 @@ public IncidentServices incidentServices() {
     return new IncidentServices(brokerClient, searchClient, transformers, authentication);
   }
 
-  public ManagementService managementService() {
```

Comment:
I took it out of `CamundaServices` since `ManagementService` will not be a part of where `CamundaServices` puts services into a java beans in `CamundaServicesConfiguration`. It can't go in `CamundaServicesConfiguration` since that config class is conditional based on is REST is enabled or not. The license information needs to be accessible in any situation. This is why I have `ManagementServicesConfiguration` making the `ManagementService` bean

My thought was that by putting it in a separate configuration class, it was more consistent. The configuration classes `CamundaServicesConfiguration`and `ManagementServicesConfiguration` are responsible solely for the services that they cover. I did not want to split where `CamundaService` is but into a bean into two different config classes